src/Translator.php

```diff
@@ -38,10 +38,10 @@ class Translator implements TranslatorInterface
      * @param string      $language Primary Language
      * @param string|null $fallback Fallback Language
      */
-    public function __construct(string $language, string $fallback = 'en')
+    public function __construct(string $language, ?string $fallback = 'en')
     {
         $this->language = $language;
-        $this->fallback = $fallback;
+        $this->fallback = $fallback ?? false;
```

Comment:
Why are you still using false here instead of `$this->fallback = $fallback ?? $this->fallback;` ?
There's always going to be an english string available so I think it should be somewhat forced

Also change the construct parameter to `?string $fallback = null`

I'd also suggest re-adding the `setLanguage` function, and simply calling it from the constructor.
That way it'd be possible to change the language during runtime (not very useful, but still)

Reply:
for the fallback, i don't commit the last change, it will be in the next push (today).
for the SetLanguage, i think there is no need of changing the language why allow it?

Reply:
i'm working on unit test now

Reply:
for the fallback i move the check for fallback===language in loading file from folder to avoid loading the same file twice, which is better